Communications.lua

```diff
@@ -79,6 +79,7 @@ function AstralComs:OnEvent(event, prefix, msg, channel, sender)
 
 	for _, obj in pairs(objs) do
 		if obj.prefix == arg then
+		if not obj.method then return end
```

Comment:
I wouldn't recommend this because if this bug happens again, it would be failing *silently* - you want to be notified if your AddOn is bugging out.

Reply:
Removed.